src/triage/component/catwalk/feature_importances.py

```diff
@@ -22,7 +23,8 @@ def _ad_hoc_feature_importances(model):
     """
     feature_importances = None
 
-    if isinstance(model, (sklearn.linear_model.logistic.LogisticRegression)):
+    if (isinstance(model, (sklearn.linear_model.logistic.LogisticRegression)) or 
+        isinstance(model, (ScaledLogisticRegression))):
```

Comment:
no opinion on the more profound stuff, just FYI, this is briefer :wink: 

```
        if isinstance(model, (ScaledLogisticRegression,
                              sklearn.linear_model.logistic.LogisticRegression)):
```

Reply:
👍